### DISCUSSION TITLE
bigfix/10.0/mcm/MCM/Install/c_google_enterprise_credentials

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#21] [Q&A]
# bigfix/10.0/mcm/MCM/Install/c_google_enterprise_credentials

Google Enterprise Credentials are needed to utilize Android Management functionalities.

https://help.hcltechsw.com/bigfix/10.0/mcm/MCM/Install/c_google_enterprise_credentials.html

**Comment by gayathripg** [comment]
test
